CHANGES.md

```diff
@@ -7,6 +7,10 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 ### What's new
 
 * The DC/OS configuration variable `mesos_seccomp_enabled` now defaults to `true`, with `mesos_seccomp_profile_name` set to `default.json`. This is not expected to break tasks. If you experience problems, though, please note that seccomp can be disabled for individual tasks through the DC/OS SDK and Marathon. (DCOS-50038)
+
 * Updated ref of dvdcli to fix dvdcli package build (DCOS-53581)
 
+
 ### Breaking changes
+
+Admin Router now requires a CPU with SSE4.2 support.
```

Comment:
Note: will remove this note in a follow-up, it's _not_ a breaking change, see JIRA ticket mentioned in PR description.

Reply:
Copying my comment from JIRA:

```
About SSE4.2, I have tested our CockroachDB binary in soak113s and it uses advanced instruction sets, too:

/elfx86exts /opt/mesosphere/active/cockroach/bin/cockroach
MODE64 (call)
CMOV (cmove)
SSE1 (xorps)
SSE2 (movdqu)
AVX (vmovdqu)
AVX2 (vpcmpeqb)
SSE42 (pcmpestri)
AES (aesenc)
SSSE3 (pshufb)
SSE41 (pinsrd)
NOVLX (vmovntdq)
FMA (vfnmadd231sd)
BMI2 (mulx)
ADX (adcx)
PCLMUL (pclmulqdq)
BMI (andn)
CPU Generation: Unknown

That is, we can actually safely build OpenResty with SSE4.2 support and it will not be a breaking change for anyone coming from 1.13.
```